environment_dev.yml

```
@@ -6,8 +6,9 @@ channels:
   - conda-forge
 dependencies:
   - python=3.10
-  - yarn>=4.4.0
-  - nodejs>=18.0
+  # Match the version of Node.js defined in the Dockerfile.
+  # Also, do not install Yarn separately, use Corepack (comes with Node.js).
+  - nodejs=18.18
```

```suggestion
  - nodejs=18.18
```
but why? package.json has `>=18.0` so this is our canonical version. So let's just bump all to `>=22.0<23.0` explicitly to avoid mismatch 

See https://endoflife.date/nodejs

---

I was thinking about doing that in a follow-up PR, hence:

- https://github.com/conda-incubator/conda-store-ui/issues/416

---

Ah, just saw your comment on that issue (#416). Since I'm already neck-deep in all of this, I'll go ahead and update Node